packages/nx/src/command-line/init/implementation/react/rename-js-to-jsx.ts

```diff
@@ -8,11 +8,14 @@ export function renameJsToJsx(appName: string, isStandalone: boolean) {
   );
 
   files.forEach((file) => {
-    const content = readFileSync(file).toString();
-    // Try to detect JSX before renaming to .jsx
-    // Files like setupTests.js from CRA should not be renamed
-    if (/<[a-zA-Z0-9]+/.test(content)) {
-      renameSync(file, `${file}x`);
+    const fileStats = lstatSync(file);
```

Comment:
We can use `fileExists` from fileutils here.

Reply:
Done